res/css/structures/_UserMenu.scss

```diff
@@ -36,7 +60,7 @@ limitations under the License.
             mask-size: contain;
             mask-repeat: no-repeat;
             background: $primary-fg-color;
-            mask-image: url('$(res)/img/element-icons/context-menu.svg');
+            mask-image: url('$(res)/img/feather-customised/chevron-down.svg');
```

Comment:
Hmm, is it intentional to change this for non-prototype users as well?

Reply:
Yes:


![image](https://user-images.githubusercontent.com/1190097/92010948-a363c480-ed07-11ea-9461-9f2f1c75163f.png)